_includes/master/entityrule.md

```diff
@@ -1,10 +1,9 @@
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
-| tag                   | Deprecated: Match on tag. |  | string | |
-| notTag                | Deprecated: Negative match on tag. |  | string | |
 | nets                  | Match packets with IP in any of the listed CIDRs. | List of valid IPv4 or IPv6 CIDRs  | list of cidrs |
 | notNets               | Negative match on CIDRs. Match packets with IP not in any of the listed CIDRs. | List of valid IPv4 or IPv6 CIDRs  | list of cidrs |
 | selector    | Positive match on selected endpoints. | Valid selector | [selector](#selector) | |
 | notSelector | Negative match on selected endpoints. | Valid selector | [selector](#selector) | |
+| namespaceSelector | Positive match on selected namespaces. If specified, only workload endpoints in the selected namespaces are matched. | Valid selector | [selector](#selector) | |
```

Comment:
This is where an explanation in the introduction which we could cross-reference here would be useful. There are a lot of namespaces out there: Kubernetes, Linux, etc. It seems everyone has a namespace and so what namespace means in this context is not going to be clear. Right now, I think we have the info what is a Calico namespace in the intro of `GlobalNetworkPolicy` and `NetworkPolicy`...it's not something we can cross-reference.

Reply:
Easy enough to say "selected kubernetes namespaces" here instead.

Reply:
Would be good to have a page on namespaces and how they interact with diff orchestrators somewhere, but the calicoctl reference might not be the right place.

Reply:
Oh, so then these are kubernetes namespaces. Yeah, then you can just say "Kubernetes namespaces" that works fine.